client/src/components/RatingsReview.jsx

```diff
@@ -1,38 +1,201 @@
+          <Component option={this.state.option} />
+        </React.Fragment>
+      )
+    }
+  }
+};
+I think we could also do this functionally:
+const higherOrderComponent = (Component) => {
+  return (props) => {
+    const [option, setOption] = useState(2);
+    handleChange = (e) => setOption(e.target.value)
+      return  (
+        <React.Fragment>
+          <input id="mood-range" type="range" min={0} max={4} value={option} onChange={this.handleChange} />
+          <Component option={option} />
+        </React.Fragment>
+      )
+  }
+};
 
-Review Tile:
 
 */
 
+var characteristicBreakdown = {
+  'Size': ['A size too small', '½ a size too small', 'Perfect', '½ a size too big', 'A size too big'],
+  'Width': ['Too narrow', 'Slightly narrow', 'Perfect', 'Slightly wide', 'Too wide'],
+  'Comfort': ['Uncomfortable', 'Slightly uncomfortable', 'Ok', 'Comfortable', 'Perfect',],
+  'Quality': ['Poor', 'Below Average', 'What I expected', 'Pretty great', 'Perfect',],
+  'Length': ['Runs Short', 'Runs slightly short', 'Perfect', 'Runs slightly long', 'Runs long'],
+  'Fit': ['Runs tight', 'Runs slightly tight', 'Perfect', 'Runs slightly long', 'Runs long']
+};
+
 
 const Ratings = (props) => {
-  const [reviews, setReviews] = useState({});
-  //useEffect to async load reviews
+  const [reviews, setReviews] = useState([]);
+  const [meta, setMeta] = useState({});
+  const [count, setCount] = useState(2);
+  const [page, setPage] = useState(1);
+  const [sort, setSort] = useState('relevant');
+
+  //effects
+  //runs when the sorting method is changed, fetch reviews with new sorting methods
   useEffect(() => {
+    setPage(1)
+    setReviews([]);
     async function fetchAPI() {
-      var res = await Atelier.getReviews(13027);
+      var res = await Atelier.getReviews(props.id, 2, 1, sort);
+      //console.log('res: ' + JSON.stringify(res));
+      return res.results;
+    }
+    fetchAPI().then(res => {
       setReviews(res);
+    });
+  }, [sort]);
+
+  //runs when load more is clicked, fetch the next page of reviews
+  useEffect(() => {
+    async function fetchAPI() {
+      var res = await Atelier.getReviews(props.id, 2, page, sort);
+      return res.results;
+    }
+    if (page > 1) {
+      fetchAPI().then(res => {
+        setReviews(reviews.concat(res));
+      });
+    }
+  }, [page]);
+
+  //runs when the component loads, fetch the review metadata
+  useEffect(() => {
+    async function fetchAPI() {
+      var res = await Atelier.getMeta(props.id);
+      setMeta(res);
     }
     fetchAPI();
-  }, []);
-  return <div>{JSON.stringify(reviews)}</div>
+  }, [props.id]);
+
+  //handle the load more button, loads the next page
+  const handleLoad = () => {
+    setPage(page + 1);
+  };
+
+  //handle changing the sort dropdown, load new reviews
+  const handleChange = (e) => {
+    setSort(e.target.value)
+  };
+
+  //render only if there has been a review
```

Comment:
Could you break this into clearer conditional rendering using state? Might be more readable. But this is cool, I didn't think you could do this.